lib/kernel-manager.js

```diff
@@ -8,20 +9,35 @@ import path from "path";
 
 import Config from "./config";
 import ZMQKernel from "./zmq-kernel";
+// I imported Kernel so it could be a valid parameter type
+import Kernel from "./kernel.js";
 import KernelPicker from "./kernel-picker";
 import store from "./store";
 import { getEditorDirectory, log } from "./utils";
 
 class KernelManager {
+  // kernelspecs seems to be have this form
+  /*
+  kernelSpec = {
+    display_name: "Existing Kernel",
+    language,
+    argv: [],
+    env: {}
+  };
+  */
+  _kernelSpecs: ?Object;
+  kernelPicker: KernelPicker;
   constructor() {
     this._kernelSpecs = this.getKernelSpecsFromSettings();
   }
 
-  startKernelFor(grammar, onStarted) {
+  startKernelFor(grammar: atom$Grammar, onStarted: (kernel: Kernel) => void) {
```

Comment:
Actually I think we should use `(kernel: ZMQKernel) => void` here. Then we can probably remove [this line](https://github.com/nteract/hydrogen/pull/832/files#diff-67b721d4904ee45505901d8e7e272c5fR13).